x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/MlNativeAutodetectIntegTestCase.java

```diff
@@ -6,6 +6,8 @@
  */
 package org.elasticsearch.xpack.ml.integration;
 
+import org.elasticsearch.action.admin.cluster.snapshots.features.ResetFeatureStateAction;
```

Comment:
These have been imported here but I don't see them used.

Reply:
I wonder why formatting didn't cause it to fail :/ It is probably not used.